src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/ThrowsTaglet.java

```diff
@@ -209,12 +209,12 @@ private Content inheritThrowsDocumentation(ExecutableElement holder,
         Utils utils = writer.configuration().utils;
         Map<List<ThrowsTree>, ExecutableElement> declaredExceptionTags = new LinkedHashMap<>();
         for (TypeMirror declaredExceptionType : declaredExceptionTypes) {
-            Input input = new DocFinder.Input(utils, holder, this,
+            var input = new DocFinder.Input(utils, holder, this,
                     utils.getTypeName(declaredExceptionType, false));
             DocFinder.Output inheritedDoc = DocFinder.search(writer.configuration(), input);
             if (inheritedDoc.tagList.isEmpty()) {
-                String typeName = utils.getTypeName(declaredExceptionType, true);
-                input = new DocFinder.Input(utils, holder, this, typeName);
+                input = new DocFinder.Input(utils, holder, this,
+                        utils.getTypeName(declaredExceptionType, true));
```

Comment:
I guess I don't understand your coding style, because elsewhere (earlier) in this review, you factored out sub-expressions into their own `var` variable.

Reply:
It was later fixed in 40ca8088.